### PR TITLE
UserGuide: update json Config File instructions

### PR DESCRIPTION
**Pull request by eugenepeh**
```
As people tend to perform trial and error for the json config file
before committing to the repository. However, the json config file will
not be included when we perform the clone on the local repository.

Let's add the instructions to request user to commit the json config
file.
```

temporary fix for #379

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Repo owners can provide the following additional information to RepoSense using
 * which git and github usernames belong to which authors
 * the display of an author
 
-To use this feature, add a `_reposense/config.json`  to the root of your repo using the format in the example below ([another example](../_reposense/config.json)):
+To use this feature, add a `_reposense/config.json`  to the root of your repo using the format in the example below ([another example](../_reposense/config.json)) and **commit it**:
```
Comment (damithc) [inline docs/UserGuide.md]: `and **commit it** (reason: RepoSense can see committed code only):`?

Comment (damithc) [inline docs/UserGuide.md]: Should we mention this under `@@author` section too?

Comment (eugenepeh) [reply docs/UserGuide.md]: Hmm... but its not really relevant because `@@author` can typically work with/without json file.

Comment (damithc) [reply docs/UserGuide.md]: But the tags will not affect the results unless they are committed first, right?

Comment (eugenepeh) [reply docs/UserGuide.md]: oh... okay

**Pull request merged**